src/core/toType.js

```diff
@@ -0,0 +1,28 @@
+define( function() {
+	"use strict";
+
+	var class2type = {
```

Comment:
We usually skip the first indentation so that almost whole files are not indented and it's easier to later migrate to ES6 modules without changing virtually all the lines.

Some smaller modules don't follow this pattern but most are. So please de-indent it all by one tab.